baseline/OSPS-QA.yaml

```
@@ -237,3 +227,65 @@ criteria:
     control_mappings:       
       BPB: B-G-3
     security_insights_value: # TODO
+
+  - id: OSPS-QA-11
+    maturity_level: 2
+    criterion: |
+      All released software assets MUST be
```

```suggestion
      All released built software assets MUST be
```

The rest of the text presumes it's built, but the criterion doesn't say so.